modules/nw-multi-network-policy-enable.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+[id="nw-multi-network-policy-enable_{context}"]
+= Enabling multi-network policy for the cluster
+
+As a cluster administrator, you can enable multi-network policy support on your cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file with the following YAML:
```

Comment:
Nothing wrong with what you have, but I'll share an idea.  You could be prescriptive and suggest creating a file named `multi-network-policy.yaml` in this step and then reference the same file name in the next.  That would eliminate the somewhat superfluous suggestion to substitute the real file name.  I imagine that customers understand they can name files whatever they want.